lib/files/node.ts

```diff
@@ -296,4 +328,34 @@ export abstract class Node {
 		}
 	}
 
+	/**
+	 * Update the attributes of the node
+	 *
+	 * @param attributes The new attributes to update
+	 */
+	update(attributes: Attribute) {
+		// Proxy the attributes to update the mtime on change
+		// eslint-disable-next-line @typescript-eslint/no-explicit-any
+		this._attributes = new Proxy(this.cleanAttributes(attributes) as any, this.handler)
+	}
+
+	private cleanAttributes(attributes: Attribute): Attribute {
```

Comment:
This is a breaking change!
This removes e.g. `size` from attributes causing the files app to fail on Nextcloud 28+.

Reply:
AH, nice to know!
Why was the attributes size used and where? :thinking: